nginx/conf.d/scalelite-ssl.template

```diff
@@ -4,9 +4,9 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
-    location / {
-        return  301 https://$host$request_uri;
-      }
+	location / {
+		return  301 https://$host$request_uri;
```

Comment:
This actually changes the formatting in a different manner than before where you used two spaces.

Reply:
Should be fixed now